<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Hello PyQt</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>601</width>
     <height>531</height>
    </rect>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>A</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>B</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>C</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>D</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>E</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_import">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Excel Import</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_export">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Excel Export</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
